accelerators/corda/service-bus-integration/service-bus-listener/src/main/kotlin/net/corda/workbench/serviceBus/TestDataSender.kt

```diff
@@ -14,7 +14,7 @@ import java.util.*
 /**
  * Build test data and sends to a queue
  */
-class TestDataSender(val queueClient: QueueClient) {
+class TestDataSender(val queueClient: ServiceBusSenderClient) {
```

Comment:
Since this is no longer a queue client, I would recommend changing the name of the variable here from `queueClient` to `senderClient`